tests/test_gds_featurizer.py

```diff
@@ -52,11 +52,11 @@ def test01_add_attribute(self):
                 pass
         except:
             pass
-        self.assertEqual( self.featurizer._add_attribute("VERTEX","FLOAT","attr1"),'Global schema change succeeded.')
+        self.assertEqual( self.featurizer._add_attribute("VERTEX","FLOAT","attr1", global_change=True),'Schema change succeeded.')
 
     def test02_add_attribute(self):
         try:
-            tasks = "ALTER Edge Trans DROP ATTRIBUTE (attr2);"
+            tasks = "ALTER Edge Cites DROP ATTRIBUTE (attr2);"
```

Comment:
Does this test run through? Cora2 does not have edge type Cites but Cite2. But I guess it is fine since Cites is a global edge type? However, it would be safer to use Cite2 here.

Reply:
Whoops, let me fix that. It did run but obviously is the wrong edge

Reply:
Resolved.